pom.xml

```diff
@@ -212,6 +213,13 @@
         <version>${version.org.assertj_assertj-core}</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
```

Comment:
Actually I was thinking about moving to explicit dep's, hence the bom. and doesn't hurt. make bom not war